content/docs/start/experiments.md

```diff
@@ -104,24 +134,6 @@ experiment in a separate row, along with the Git commit IDs they are attached
 to. We can see that the experiment we run has a name `exp-6dccf` and was run
 from the commit ID `7317bc6`.
 
-<details>
-
-### ℹ️ If you used `dvc repro` before
-
-Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
-have a DVC project, you may already be using `dvc repro`.
-
-We use `dvc repro` to run the pipeline as found in the <abbr>workspace</abbr>.
-All the parameters and dependencies are retrieved from the current workspace. It
-doesn't use any specialized mechanism to track experiments.
-
-When you have a large number of experiments that you don't want to commit all to
-Git, it's better to use `dvc exp run`. It allows to change the parameters
-quickly, can track the history of artifacts and has facilities to compare these
-experiments easily.
-
-</details>
-
 Now let's do some more experimentation.
 
 DVC allows to update the parameters defined in the pipeline without modifying
```

Comment:
I think it makes sense to remove the `repro` section for sure. But then we go and mention "the pipeline" out of the blue right after. What is a pipeline? My first thought is to also remove that reference but maybe linking to https://dvc.org/doc/start/data-pipelines instead could be an option (since we're in the GS and there's no specific pipelines guide anyway).

Reply:
I replaced two occurrences of "pipeline" to "project."